strongbox-web-core/src/test/java/org/carlspring/strongbox/forms/users/UserFormTestIT.java

```diff
@@ -156,8 +154,8 @@ void testUserFormExistingUserInvalidPasswordLength()
         Set<ConstraintViolation<UserForm>> violations = validator.validate(userForm, UserForm.ExistingUser.class);
 
         // then
-        assertFalse(violations.isEmpty(), "Violations are empty!");
-        assertEquals(violations.size(), 1);
+        assertThat(violations.isEmpty()).as("Violations are empty!").isFalse();
+        assertThat(violations.size()).isEqualTo(1);
```

Comment:
```       
assertThat(violations.isEmpty()).as("Violations are empty!").isFalse();
assertThat(violations.size()).isEqualTo(1);
```
=>
```
assertThat(violations).hasSize(1);
```

Reply:
Please correct it everywhere where possible with `.size()`

Reply:
Also, @mslowiak , if you see two or more assertions that lead to the same result, you can simplify it as proposed above.